presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -45,3 +45,101 @@ which is configured with a carefully selected set of options. This behavior
 can be overridden by setting the environment variable ``PRESTO_PAGER`` to the
 name of a different program such as ``more`` or `pspg <https://github.com/okbob/pspg>`_,
 or it can be set to an empty value to completely disable pagination.
+
+Output formats
+--------------
+
+The Presto CLI provides the option ``--output-format`` to control how the output is
+displayed when running in noninteractive mode. The available options shown in the
+following table must be entered in uppercase. The default value is ``CSV``.
+
+.. list-table::
+  :widths: 20, 80
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``CSV``
+    - Comma-separated values, each value quoted. No header row.
+  * - ``CSV_HEADER``
+    - Comma-separated values, quoted with header row.
+  * - ``CSV_UNQUOTED``
+    - Comma-separated values without quotes.
+  * - ``CSV_HEADER_UNQUOTED``
+    - Comma-separated values with header row but no quotes.
+  * - ``TSV``
+    - Tab-separated values.
+  * - ``TSV_HEADER``
+    - Tab-separated values with header row.
+  * - ``JSON``
+    - Output rows emitted as JSON objects with name-value pairs.
+  * - ``ALIGNED``
+    - Output emitted as an ASCII character table with values.
+  * - ``VERTICAL``
+    - Output emitted as record-oriented top-down lines, one per value.
+  * - ``NULL``
+    - Suppresses normal query results. This can be useful during development
+      to test a query's shell return code or to see whether it results in
+      error messages.
+
+Examples
+^^^^^^^^
+
+Consider the following command run as shown, or with ``--output-format CSV``:
+
+.. code-block:: none
+
+   presto --execute 'SELECT nationkey, name, regionkey FROM tpch.sf1.nation LIMIT 3'
```

Comment:
; at end of this statement is not necessary but maybe @electrum prefers that since we do that for other query examples

Reply:
This is a `--execute` argument which doesn't need once since it's a single statement to be executed. The command can be run-as by copy/paste.

The reason to add a semicolon to SQL example blocks is so that they can be copy/pasted directly, and to make it clear they are a full statement.